xml/System.Collections/ArrayList.xml

```diff
@@ -56,7 +56,11 @@
       <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- The <xref:System.Collections.ArrayList> class is designed to hold heterogeneous collections of objects. However, it does not always offer the best performance. Instead, we recommend the following:  
+
+>  [!IMPORTANT]
+>  We don't recommend that you use the `ArrayList` class for new development. Instead, we recommend that you use the generic <xref:System.Collections.Generic.List%601> class. For more information, see [Non-generic collections shouldn't be used](https://github.com/dotnet/platform-compat/blob/master/docs/DE0006.md) on GitHub.
+
+The <xref:System.Collections.ArrayList> class is designed to hold heterogeneous collections of objects. However, it does not always offer the best performance. Instead, we recommend the following:  
```

Comment:
With this change, there is now a note recommending to use `List<T>`, followed by another recommendation to use `List<T>`. Should these two be combined? Or at least connected somehow?

Reply:
Yes, it should be, @svick. Thanks for pointing that out. I should have read the documentation for ArrayList more carefully before adding the note.